sum-app.js

```diff
@@ -1,3 +1,31 @@
 class SumApp {
   // Add you solution here
+  constructor(number) {
+    this.numbers = [];
+    this.sum = 0;
```

Comment:
`this.sum` als `let sum` in der `getSum()`-Methode deklarieren. Hier kann es sonst zu Fehlern kommen (edge cases)